continuous-integration/README.md

```diff
@@ -0,0 +1,27 @@
+# Continuous Testing Guide
```

Comment:
This file needs to be updated to reflect the "Integration" guide, not "Testing".

Reply:
Done.